connector-runtime/connector-runtime-core/src/test/java/io/camunda/connector/runtime/core/outbound/ConnectorJobHandlerTest.java

```diff
@@ -806,6 +807,33 @@ void shouldCreateBpmnError_UsingExceptionCodeAndRawContext() {
       assertThat(result.getErrorMessage()).isEqualTo("Message: exception message");
     }
 
+    @Test
+    void shouldCreateBpmnError_UsingExceptionCodeAndErrorVariables() {
+      // given
+      var errorExpression =
+          "if error.code != null then "
+              + "{ \"errorType\": \"bpmnError\", \"code\": error.code, \"message\": \"Message: \" + error.message, \"variables\": error.errorVariables} "
+              + "else {}";
```

Comment:
Could you also add a test that verifies that the error variables are successfully passed to the command in https://github.com/camunda/connectors/blob/e277b0f002987e992d16a1384a9e167e381cbe10/connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/outbound/ConnectorJobHandler.java#L289 when there is no error expression?

Reply:
fixed